source/ut_assert.pkb

```diff
@@ -145,6 +145,37 @@ create or replace package body ut_assert is
   begin
     build_assert_result(a_condition, 'this', 'boolean', 'boolean', ut_utils.to_string(true), ut_utils.to_string(a_condition), ut_utils.to_string(a_msg));
   end;
+	
+  -- Strings assertions
+  procedure str_like(a_msg in varchar2, a_checking_string in varchar2, a_mask in varchar, a_escape_char in varchar2) is
```

Comment:
maybe `is_like` or `is_matching` instead of `str_like` and `a_actual` instead of `a_checking_string`, to keep it consistent with the rest of asserts.